multi_organ_segmentation/docs/README.md

```diff
@@ -0,0 +1,94 @@
+# Multi-organ segmentation in abdominal CT
+
+### **Authors**
+
+Chen Shen<sup>1</sup>, Holger R. Roth<sup>2</sup>, Kazunari Misawa<sup>3</sup>, Kensaku Mori<sup>1</sup>
+
+1. Nagoya University, Japan
+
+2. NVIDIA Corporation, USA
+
+3. Aichi Cancer Center, Japan
+
+### **Tags**
+
+Segmentation, Multi-organ, Abdominal
+
+## **Model Description**
+
+This model uses the DiNTS model architecture searched on [Medical Segmentation Decathlon](http://medicaldecathlon.com/) Pancreas [1] and re-trained for multi-organ segmentation from abdominal CT images [2,3].
+
+## **Data**
+
+This model was trained on an abdominal CT dataset in portal venous phase collected from Aichi Cancer Center in Japan. Since this is a private dataset, similar models can be trained using other public multi-organ datasets like [BTCV](https://www.synapse.org/#!Synapse:syn3193805/wiki/89480).
+
+For this bundle, we split the 420 cases into training, validation and testing with 300, 60 and 60 cases, respectively.
+
+The labels are {"0": "background","1": "artery","2": "portal vein","3": "liver","4": "spleen","5": "stomach","6": "gallbladder","7": "pancreas"}
```

Comment:
Here could the format of labels be changed? It may not need to use a dict based form in readme, thanks!
An example of the brats bundle is: https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation

Reply:
I revised this problem.